src/client/pages/OfferNew/Checkout/UserDetailsForm.tsx

```diff
@@ -142,11 +145,13 @@ export const UserDetailsForm: React.FC<Props> = ({
         inputMode="numeric"
         pattern="[0-9]*"
         maxLength={ssnMaxLength}
+        onFocus={() => setIsShowingCreditCheckInfo(true)}
         value={ssn}
         errors={ssnBackendError ? textKeys[ssnBackendError]() : undefined}
         onChange={handleSsnChange}
         onBlur={handleSsnBlur}
       />
+      {useCreditCheckInfo() && isShowingCreditCheckInfo && <CreditCheckInfo />}
```

Comment:
I just a little confused to see a hook called in the return statement. Maybe move it up and assign it to a variable first?


```
const hasEnabledCreditCheckInfo = useCreditCheckInfo()
```

Reply:
Done